### PR TITLE
feature: support TLS connection with etcd.

### PR DESCRIPTION
**Pull request by tokers**

### CODE REVIEW

**Comment by tokers** [comment]
@membphis

**Comment by membphis** [comment]
rerun the test cases right now, the CI is failed before.

File: lib/resty/etcd/v2.lua [hunk 1]
```diff
@@ -47,6 +47,7 @@ function _M.new(opts)
     local user = opts.user
     local password = opts.password
     local serializer = opts.serializer
+    local ssl_verify = opts.ssl_verify
```
Comment (membphis) [inline lib/resty/etcd/v2.lua:50]: we need to update the doc.

File: utils/check-lua-code-style.sh [hunk 2]
```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
```
Comment (membphis) [inline utils/check-lua-code-style.sh:1]: why do we need to delete this script?

one PR for one thing.

**Review by membphis — CHANGES_REQUESTED** [review]

File: lib/resty/etcd/v2.lua (hunk 1 above)
Comment (tokers) [reply lib/resty/etcd/v2.lua:50]: OK

File: utils/check-lua-code-style.sh (hunk 2 above)
Comment (tokers) [reply utils/check-lua-code-style.sh:1]: Sorry, it's an accident.

**Review by membphis — APPROVED** [review]

**Comment by membphis** [comment]
@tokers  do we need to release a new version now?

**Pull request merged**